test/unit/test_errors.py

```diff
@@ -0,0 +1,41 @@
+import os
```

Comment:
add copyright header

Reply:
I think we discussed this already - none of the files there have one, so I didn't add one either to be consistent. Did we decide they all need one? I don't remember.

Reply:
OK lets deal with that issue separately.